src/main/kotlin/fi/metatavu/timebank/api/impl/translate/PersonsTranslator.kt

```diff
@@ -16,11 +17,15 @@ class PersonsTranslator: AbstractTranslator<ForecastPerson, Person>() {
     @Inject
     lateinit var keycloakController: KeycloakController
 
+    @Inject
+    lateinit var vacationUtils: VacationUtils
 
-    override fun translate(entity: ForecastPerson): Person {
+    override suspend fun translate(entity: ForecastPerson): Person {
         val keycloakUser = keycloakController.findUserByEmail(entity.email)
         val minimumBillableRate = if (keycloakUser == null) 75 else keycloakController.getUsersMinimumBillableRate(keycloakUser)
```

Comment:
Move the default minimum billable rate (75) to a static constant within `PersonsController`.

In Kotlin, static class members/methods are defined as
```kotlin
class PersonsController {
    ...
    companion object {
        const val DEFAULT_MINIMUM_BILLABLE_RATE = 75
    }
}
```

That would then be accessed, for example in here as `PersonsController.DEFAULT_MINIMUM_BILLABLE_RATE`.